### ISSUE TITLE
0.14.1 release

### ISSUE THREAD

**Issue by eitsupi** [pola-rs/r-polars#820]
@etiennebacher What do you think it might be worth releasing 0.14.1 once #597 is merged?

**Comment by etiennebacher** [comment]
Fine for me, but #819 could be included too

**Comment by etiennebacher** [comment]
Since it will be super easy to add the subnamespaces for Series after #819, I think we should add them in a separate PR before 0.14.1.

**Comment by eitsupi** [comment]
Once #831 is merged, I will create the release.

**Comment by eitsupi** [comment]
Done in #837
